examples/aws-cdk/package.json

```
@@ -0,0 +1,27 @@
+{
+  "license": "Apache-2.0",
```

Please use the correct license from the main package

```suggestion
  "license": "MIT",
```